dev-requirements.txt

```diff
@@ -0,0 +1,9 @@
+black~=22.10
```

Comment:
could you call this file `requirements-dev.txt` doing so the requirements are close to each other and easily findable.

Reply:
I called it the same way it was in the dbt adapter template.

Reply:
ah ok ok, I generally call it `requirements-dev` in my projects, but all right, let's be alligned as much as possible to other adatpers.